core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -292,22 +289,6 @@ class Analyzer(
       sender ! VoidResponse
   }
 
-  def handleReloadFiles(files: List[SourceFileInfo]): RpcResponse = {
-    val (existing, missingFiles) = files.partition(_.exists())
```

Comment:
Not sure what do you mean, `handleReloadFiles` was just moved to `RichPresentationCompiler.scala` but is left intact.